source/api.rst

```diff
@@ -299,14 +299,6 @@ Each returned message contains an error message and a transaction hash.
 
     <a href="https://yarnpkg.com/lang/en/" target="_blank">yarn</a>
 
-.. |catapult-service-bootstrap| raw:: html
-
-   <a href="https://github.com/nemtech/catapult-service-bootstrap" target="_blank">Catapult Service Bootstrap</a>
-
-.. |catapult-server| raw:: html
```

Comment:
why catapult-server has been removed? the server is one of the components of the network (the peers, API, broker)

Reply:
Because nobody was referencing it :/
I'll add it back if the text talks about it.